datalad/local/wtf.py

```diff
@@ -463,8 +463,18 @@ def __call__(*, dataset=None, sensitive=None, sections=None, flavor="full", deco
                 raise ValueError(flavor)
 
         for s in sections:
-            infos[s] = section_callables[s]()
-
+            try:
+                infos[s] = section_callables[s]()
+            except KeyError:
```

Comment:
I know I am annoying, but could there be basic unittest invocation of wtf with incorrect section and getting impossible status -- should be 1 line, right?

Asking because those errors where "while handling this exception , such exception happened" are really annoying UX ;)

Reply:
good call, done